falcon/routing/converters.py

```diff
@@ -132,8 +155,32 @@ def convert(self, value):
             return None
 
 
+class PathConverter(BaseConverter):
+    """Field converted used to match the rest of the path.
+
+    This field converter matches the remainder of the URL path,
+    returning it as a string.
+
+    This converter is currently supported only when used at the
+    end of the url template.
+
+    The classic routing rules of falcon apply also to this converter:
```

Comment:
Given this is documented for the developer, I'm OK with leaving the behavior as-is.